integration/integration_dummy_test.go

```diff
@@ -25,17 +26,19 @@ func TestDummyIntegration(t *testing.T) {
 	dir = filepath.Dir(dir)
 
 	req := testcontainers.ContainerRequest{
-		Image:      "node:lts-buster",
-		Cmd:        []string{"tail", "-f"},
-		BindMounts: map[string]string{dir: "/data"},
+		Image: "node:lts-buster",
+		Cmd:   []string{"tail", "-f"},
+		Mounts: testcontainers.Mounts(
+			testcontainers.BindMount(dir, "/data"),
+		),
```

Comment:
modification needed due to changes introduced in the new version of the `testcontainers`